zk-sdk/src/encryption/mod.rs

```diff
@@ -10,6 +10,9 @@
 //! - Basic type-wrapper around the AES-GCM-SIV symmetric authenticated encryption scheme
 //! implemented by [aes-gcm-siv](https://docs.rs/aes-gcm-siv/latest/aes_gcm_siv/) crate.
 
+#[cfg(not(target_os = "solana"))]
+#[macro_use]
```

Comment:
Sorry, looking through this again, I don't think you need `#[macro_use]` either

Reply:
Oh oops, yeah I guess this is only required by old rust. Thanks! I will removed on a follow-up.

Reply:
Hm, actually it seems like if I removed this `macro_use` then the macros seem to be not accessible to other crates. Reading a bit on the `macro_use` attribute, it seems like this is required here?

Reply:
Ah ok, I thought the macros were only used internally. no worries then!